common/constants.go

```diff
@@ -60,7 +60,7 @@ const (
 	MinLongPollTimeout = time.Second * 2
 	// CriticalLongPollTimeout is a threshold for the context timeout passed into long poll API,
 	// below which a warning will be logged
-	CriticalLongPollTimeout = time.Second * 20
+	CriticalLongPollTimeout = time.Second * 10
```

Comment:
Changed this since, it's useful SDK side, to reduce the timeout while under load (so that when load runs out, we scale down faster).

Talked w/ David and it seems like reducing this ought to be fine, but we can undo if any objections.